### ISSUE TITLE
dashboard does not reflect reviewer changes

### ISSUE THREAD

**Issue by danielskatz** [openjournals/joss#468]
This is not really a big deal, but in my dashboard now, there's a link to a [paper](http://joss.theoj.org/papers/9cce385bc676b84853e8026ac452eb33) that shows the reviewers as the old reviewers.  In the midst of the review, I made a change of reviewers which does not show up in the dashboard.

<img width="1274" alt="screen shot 2018-12-01 at 6 12 27 am" src="https://user-images.githubusercontent.com/2913845/49328089-2a32ae80-f530-11e8-853c-a99fb3704c4c.png">

**Comment by arfon** [comment]
Yes, I knew about this one. We basically don't update the JOSS dashboard if reviewers are changed mid-review. I've marked this as both a bug, and an enhancement :-)
